package.json

```diff
@@ -0,0 +1,183 @@
+{
+  "name": "ngx-feature-flag",
+  "version": "0.1.0-development",
+  "description": "feature flag component and services for angular applications",
+  "main": "index.js",
+  "module": "index.js",
+  "typings": "index.d.ts",
+  "scripts": {
+    "build": "npm run remove-dist && gulp build:library",
+    "build:demo": "webpack --config config/webpack.demo.js --progress --profile --bail",
+    "clean": "rimraf node_modules doc coverage dist dist-demo bundles",
+    "bundle-webpack": "webpack --config config/webpack.prod.js --progress --profile --bail",
+    "reinstall": "npm run clean && npm install",
+    "remove-dist": "rimraf dist",
+    "start": "webpack-dev-server --config config/webpack.demo.js --progress --host 0.0.0.0 --port 3001 --profile --watch --content-base src/",
+    "start:demo": "webpack-dev-server --config config/webpack.demo.js --progress --host 0.0.0.0 --port 8001 --profile --watch --content-base dist-demo",
+    "test": "npm run test:unit",
+    "test:unit": "karma start",
+    "test:debug": "karma start --no-single-run --browsers Chrome",
+    "semantic-release": "semantic-release pre && npm run build && cp -r .git dist && npm publish dist"
+  },
+  "license": "Apache-2.0",
+  "contributors": [
+    {
+      "name": "Corinne Krych",
+      "email": "ckrych@redhat.com"
+    }
+  ],
+  "keywords": [
+    "angular",
+    "angular2",
+    "angular 2",
+    "angular4",
+    "angular 4",
+    "ng",
+    "ng2",
+    "ng4",
+    "feature-flag",
+    "ng-feature-flag",
+    "ngx-feature-flag"
+  ],
+  "homepage": "https://github.com/fabric8-ui/ngx-feature-flag",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fabric8-ui/ngx-feature-flag.git"
+  },
+  "bugs": {
+    "url": "https://github.com/fabric8-ui/ngx-feature-flag/issues"
+  },
+  "engines": {
+    "node": ">= 8.3.0",
+    "npm": ">= 5.3.0"
+  },
+  "dependencies": {
+    "lodash": "^4.17.10",
+    "ngx-base": "^2.3.2",
```

Comment:
anything in a peer dependency should not be in `dependencies` but should be in `devDependencies`